.github/workflows/business-ar-reminder-cd.yaml

```diff
@@ -0,0 +1,30 @@
+name: Business AR Reminder CD
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - "jobs/**"
```

Comment:
do you want to have a single workflow for all jobs under jobs/, because this will redeploy all jobs in this directory, even only one of them changes. notice how this is done here: https://github.com/bcgov/sbc-pay/blob/630a9a81c8177ddb575deab776f824c7791c93d7/.github/workflows/payment-jobs-cd.yml#L8 and the corresponding location of the devops folder https://github.com/bcgov/sbc-pay/tree/main/jobs/payment-jobs/devops

Reply:
very helpful instructions, just updated